### PR TITLE
Updating the documentation of the redis staticIP property

### PR DESCRIPTION
**Pull request by TimLovellSmith**
It is no longer required for redis VNET create scenarios, nor even recommended.

# Conflicts:
#	specification/redis/resource-manager/Microsoft.Cache/stable/2016-04-01/redis.json

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [n/a] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/redis/resource-manager/Microsoft.Cache/stable/2015-08-01/redis.json [hunk 1]
```diff
@@ -451,7 +451,7 @@
         },
         "staticIP": {
           "type": "string",
-          "description": "Required when deploying a Redis cache inside an existing Azure Virtual Network."
+          "description": "Deprecated. May optionally be used to request a specific IP address, only when deploying a Redis cache inside an Azure virtual network. We recommend you do not set this parameter, and connect using the hostname instead."
```
Comment (yungezz) [inline specification/redis/resource-manager/Microsoft.Cache/stable/2015-08-01/redis.json]: hi @TimLovellSmith just to double confirm, is the deprecation happens in the `2015-08-01` service tag or newer version?

**Comment by yungezz** [comment]
add code owner @siddharthchatrolams to help review

**Comment by TimLovellSmith** [comment]
@yungzz We need to update ownership - @SiddharthChatrolaMs doesn't work on this any more.
/cc @JonCole

File: specification/redis/resource-manager/Microsoft.Cache/stable/2015-08-01/redis.json (hunk 1 above)
Comment (TimLovellSmith) [reply specification/redis/resource-manager/Microsoft.Cache/stable/2015-08-01/redis.json]: While the property still usually works, we would prefer to change our guidance to be for people to prefer not to use this property with all api versions.

**Comment by TimLovellSmith** [comment]
@yungezz AutoPR seems to have made some breaking changes to Java:
https://github.com/test-repo-billy/azure-sdk-for-java/pull/725/files
What should we do about that?

**Comment by yungezz** [comment]
> @yungezz AutoPR seems to have made some breaking changes to Java:
> https://github.com/test-repo-billy/azure-sdk-for-java/pull/725/files
> What should we do about that?

hi @NullMDR  could you pls help to look at the java autoPR? is it testing or a real problem? thanks.

**Comment by yungezz** [comment]
> @yungezz We need to update ownership - @SiddharthChatrolaMs doesn't work on this any more.
> /cc @JonCole

hi @TimLovellSmith , pls just update create a PR on ./CODEOWNERS file to current code owners.

**Comment by NullMDR** [comment]
@yungezz @TimLovellSmith It's for testing now. Please ignore.

File: specification/redis/resource-manager/Microsoft.Cache/stable/2015-08-01/redis.json (hunk 1 above)
Comment (yungezz) [reply specification/redis/resource-manager/Microsoft.Cache/stable/2015-08-01/redis.json]: does that mean you'll update description of all versions ?

Comment (TimLovellSmith) [reply specification/redis/resource-manager/Microsoft.Cache/stable/2015-08-01/redis.json]: Yes, the description in all versions is being updated as part of this PR.

**Review by yungezz — APPROVED** [review]

**Comment by yungezz** [comment]
hi @TimLovellSmith is this PR ready to merge? or you'll push more commits?

**Comment by yungezz** [comment]
@TimLovellSmith ping again..

**Comment by yungezz** [comment]
HI @TimLovellSmith is this PR ready to merge?

**Comment by TimLovellSmith** [comment]
No, not quite ready it turns out. I'll close for now and reopen later with some revised changes.

**Pull request closed without merging**